haveaseat-api/haveaseat-api/Controllers/ReservationController.cs

```diff
@@ -0,0 +1,31 @@
+using haveaseat.DTOs;
+using haveaseat.Repositories.Interfaces;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+
+namespace haveaseat_api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class ReservationController : ControllerBase
+{
+    private readonly IReservationRepository _reservationRepository;
+
+    public ReservationController(IReservationRepository reservationRepository)
+    {
+        _reservationRepository = reservationRepository;
```

Comment:
to nie błąd a taka ciekawostka, że mozesz wstrzykiwac service w konstruktorze od jakiegos czasu 
 coś takiego
public class ReservationController(IReservationRepository  reservationRepository)  : ControllerBase

wtedy nie musisz robic dodatkowej zmiennej i przypisywac jej w konstruktorze